### PR TITLE
Updated grafana.md to include the new InfluxDB data source SQL setup

### PR DESCRIPTION
**Pull request by Jayclifford345**
Draft for proposed Grafana changes. 

- [x] Serverless
- [x] Dedicated
- [x] Clustered

---

- [x] Signed the [InfluxData CLA](https://www.influxdata.com/legal/cla/)
  ([if necessary](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#sign-the-influxdata-cla))
- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/cloud-serverless/process-data/visualize/grafana.md [hunk 1]
```diff
@@ -126,43 +87,34 @@ query {{% product-name %}}:
...
-
-6.  Add connection **MetaData**.
-    {{% product-name %}} requires _one_ of the following key-value pairs:
-    
-    - **Key**: `database`, **Value**: Bucket name
-    - **Key**: `bucket-id`, **Value**: Bucket ID
+2.  Under **InfluxDB Details**:
 
-7.  Click **Save & test**.
+    - **Database**: Provide a database name to query. Use the database name that is mapped to your InfluxBD Cloud bucket.
```
Comment (jstirnaman) [inline content/influxdb/cloud-serverless/process-data/visualize/grafana.md:103]:
```suggestion
    - **Database**: Provide a database name to query. Use the database name that is mapped to your InfluxDB Cloud bucket.
```

File: content/influxdb/cloud-serverless/process-data/visualize/grafana.md [hunk 2]
```diff
@@ -185,17 +132,22 @@ To query {{% product-name %}} with InfluxQL, first map database and retention po
...
+2.  Under **InfluxDB Details**:
 
     - **Database**: Provide a database name to query. Use the database name that is mapped to your InfluxBD Cloud bucket.
     - **User**: Provide an arbitrary string.
       _This credential is ignored when querying {{% product-name %}}, but it cannot be empty._
     - **Password**: Provide an [API token](/influxdb/cloud-serverless/admin/tokens/)
       with read access to the buckets you want to query.
+    - **HTTP Method**: Choose one of the available HTTP request method to use\
+      when querying data:
+
```
Comment (jstirnaman) [inline content/influxdb/cloud-serverless/process-data/visualize/grafana.md:144]:
```suggestion
    - **HTTP Method**: Choose one of the available HTTP request methods to use when querying data:

```

**Review by jstirnaman — CHANGES_REQUESTED** [review]
Some minor fixes

**Comment by sanderson** [comment]
I'm holding off on the v1-v2 changes for another branch. I think it's better to get the v3 stuff out there.

**Pull request merged**